Runtime/MeshSimplifier.cs

```diff
@@ -1675,6 +1742,84 @@ private void CalculateSubMeshOffsets()
             }
         }
         #endregion
+
+        #region Traingle helper functions
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private HashSet<Triangle> GetTrianglesContainingVertex(Vertex toCheck)
+        {
+            int trianglesCount = toCheck.tcount;
+            int startIndex = toCheck.tstart;
+
+            HashSet<Triangle> tris = new HashSet<Triangle>();
+
+            for (int a = startIndex; a < startIndex + trianglesCount; a++)
+            {
+                tris.Add(triangles[refs[a].tid]);
+            }
+
+            return tris;
+        }
+
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private HashSet<Triangle> GetTrianglesContainingBothVertices(Vertex vertex1, Vertex vertex2)
+        {
+            HashSet<Triangle> tris = new HashSet<Triangle>();
+
+
+            int trianglesCount = vertex1.tcount;
+            int startIndex = vertex1.tstart;
+
+            for (int a = startIndex; a < startIndex + trianglesCount; a++)
+            {
+                Triangle tri = triangles[refs[a].tid];
+
+                Vertex v1 = vertices[tri.v0];
+                Vertex v2 = vertices[tri.v1];
+                Vertex v3 = vertices[tri.v2];
+
+                int hashcode = vertex2.GetHashCode();
+
+                if (v1.GetHashCode().Equals(hashcode) || v2.GetHashCode().Equals(hashcode) || v3.GetHashCode().Equals(hashcode))
+                {
+                    tris.Add(tri);
+                }
+
+            }
+
+
+
+            return tris;
+        }
+
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private Vector3 GetTriangleNormalVector(Triangle triangle)
```

Comment:
Which then means that this method is no longer needed.
See comment above for context.